src/api/controllers/articleController.js

```diff
@@ -232,8 +232,9 @@ export default class ArticleController {
       description: description || req.Existing.description,
       body: body || req.Existing.body,
       category: category || req.Existing.category,
-      coverImage: coverImage || req.Existing.coverImage
+      coverImage
```

Comment:
@truestbyheart why did you edit this line?

Reply:
Oooh... I was trying to troubleshoot why the image was not being uploaded so I forgot to bring it back let me put it back